### PR TITLE
Add additional information on reading the root cert and fix supported certificate managers

### PR DESCRIPTION
**Pull request by trstringer**
This PR adds additional information on reading the root cert contents (for example, to look at the certificate metadata). This also fixes the supported certificate managers documentation.

### CODE REVIEW

File: content/docs/guides/certificates.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ data:
...
 
+To read the root certificate, you can decode the secret and pipe it through openssl:
+
+```console
+kubectl get secret -n $osm_namespace osm-ca-bundle -o jsonpath='{.data.ca\.crt}' |
+    base64 -d |
+    openssl x509 -text -noout
+```
+
+This will provide valuable certificate information, such as the expiration date and the issuer.
```
Comment (shashankram) [inline content/docs/guides/certificates.md:53]: The CA bundle secret is likely not created for Hashi Vault, so this should be noted. Moreover, the CA secret may have a different name than `osm-ca-bundle`.

I'd also updateL45 to avoid using words like "pipe it through openssl`, and rephrase it to "To decode the root CA certificate, ..."

File: content/docs/guides/certificates.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ data:
 
 For details and code where this is used see [osm-controller.go](https://github.com/openservicemesh/osm/blob/release-v0.9/cmd/osm-controller/osm-controller.go#L182-L183).
 
+To read the root certificate (with the exception of Hashicorp Vault), you can retrieve the corresponding secret and decode it:
+
+```console
+kubectl get secret -n $osm_namespace $osm_ca_bundle -o jsonpath='{.data.ca\.crt}' |
```
[the remark this answers is not in the document]
Comment (nshankar13) [reply content/docs/guides/certificates.md:48]: There are some places in the docs where we have it (ex: line 63 of this certificate doc, egress_policy, permissive_traffic_mode demo, content/docs/getting_started/upgrade), but some docs don't follow this. For the sake of consistency, I think we should decide whether to have `export` throughout or scrap it altogether. 

Looking back, I personally don't see that much value in having it so maybe we can remove it

[the remark this answers is not in the document]
Comment (shashankram) [reply content/docs/guides/certificates.md:48]: If you are using bash variables, then please set them as well. I think it's easiest to use defaults such as `osm-system`, `osm-ca-bundle`, and leave a `> Note:` indicating these must be changed if using non-default values.
